express.js

```diff
@@ -318,36 +320,76 @@ app.get("/tracks/:id/:time", cors(), async (req, res) => {
 
 app.post("/imageUpload", upload.single("file"), async (req, res) => {
   //Help from https://stackoverflow.com/questions/36477145/how-to-upload-image-file-and-display-using-express-nodejs
-  let file = __dirname + "\\tmp\\" + req.file.originalname;
-  fs.rename(req.file.path, file, function (err) {
-    if (err) {
-      console.log(err);
-      res.send(500);
-    } else {
-      gm()
-        .command("convert")
-        .in(file)
-        .define("jpeg:extent=256kb")
-        .stream(function (err, stdout, stderr) {
-          if (err) {
-            console.log("ERROR: ", err);
-          } else {
-            let writeStream = fs.createWriteStream(
-              __dirname + "\\tmp\\playlistImg.jpg"
-            );
-            writeStream.on("error", function (e) {
-              console.log("ERR: ", e);
-            });
-            stdout.pipe(writeStream);
-            writeStream.on("finish", function () {
-              res.sendFile(__dirname + "\\tmp\\playlistImg.jpg");
+  req.params.id = 'spotify:dragonguy128';
```

Comment:
can this be done without hard-coding the user ID?

Reply:
That was for testing it out. It's still a work in progress but I don't think the Spotify API works for the playlist image